include/boost/beast/http/detail/rfc7230.hpp

```diff
@@ -352,8 +352,8 @@ increment()
         {
             it = first;
         };
-    v.first.clear();
-    v.second.clear();
+    v.first = string_view();
+    v.second = string_view();
```

Comment:
Consider this instead:
```
v.first = {};
v.second = {};
```

Reply:
Actually, I'll make this change myself in its own commit.

Reply:
:+1: